cristin-project/src/main/java/no/unit/nva/cristin/projects/CristinProjectBuilder.java

```diff
@@ -47,8 +49,10 @@ public CristinProject build() {
         cristinProject.setPopularScientificSummary(extractSummary(nvaProject.getPopularScientificSummary()));
         cristinProject.setProjectFundingSources(extractFundings(nvaProject.getFunding()));
         cristinProject.setParticipants(extractContributors(nvaProject.getContributors()));
-        cristinProject.setCoordinatingInstitution(
-                new CristinOrganizationBuilder(nvaProject.getCoordinatingInstitution()).build());
+        if (Objects.nonNull(nvaProject.getCoordinatingInstitution())) {
+            cristinProject.setCoordinatingInstitution(
```

Comment:
I think coordinating institution is required to create in Cristin

Reply:
Yes, it is already checked in validation on input, removed redundant check here